### PR TITLE
♻️ New fakes in dev/catalog/service

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Following up with @odeimaiz the `/dev/catalog/services` fakes will refer to sleepers

## Related issue/s
 - #5964 

## How to test
```
cd packages/models-library
make install-dev
pytest -vv tests/test__model_examples.py
```

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]
thanks

**Review by bisgaard-itis — APPROVED** [review]
Cool!

File: packages/models-library/src/models_library/api_schemas_webserver/catalog.py [hunk 1]
```diff
@@ -100,18 +99,137 @@ class Config(_BaseCommonApiExtension.Config):
...
+
+_EXAMPLE_SLEEPER: dict[str, Any] = {
+    "name": "sleeper",
+    "thumbnail": None,
+    "description": "A service which awaits for time to pass, two times.",
+    "classifiers": [],
+    "quality": {},
+    "accessRights": {"1": {"execute_access": True, "write_access": False}},
+    "key": "simcore/services/comp/itis/sleeper",
+    "version": "2.2.1",
```
Comment (sanderegg) [inline packages/models-library/src/models_library/api_schemas_webserver/catalog.py:122]: we use the sleepers 1.0.0 in a lot of tests, it is a good representative of the very old services.

**Review by sanderegg — APPROVED** [review]

File: packages/models-library/src/models_library/api_schemas_webserver/catalog.py (hunk 1 above)
Comment (pcrespov) [reply packages/models-library/src/models_library/api_schemas_webserver/catalog.py:122]: In this context is not relevant.

**Pull request merged**